src/main/resources/com/owlcyberdefense/mil-std-2045/xsd/milstd2045.internal.dfdl.xsd

```diff
@@ -142,4 +155,457 @@ SOFTWARE.
     </sequence>
   </group>
 
+
```

Comment:
This material was all added by combining milstd2045.internal_PI.dfdl.xsd and this file together.